### PR TITLE
Update to latest Brave and Zipkin versions

### PR DESCRIPTION
**Pull request by llinder**

### CODE REVIEW

File: brave-kafka/pom.xml [hunk 1]
```diff
@@ -56,6 +56,10 @@
       <groupId>io.zipkin.brave</groupId>
       <artifactId>brave</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.zipkin.java</groupId>
```
Comment (llinder) [inline brave-kafka/pom.xml]: Without this dependency compilation fails with:
```
INFO] Compiling 11 source files to /Users/llinder/git/smartoss/smartthings-brave/brave-kafka/target/classes
/Users/llinder/git/smartoss/smartthings-brave/brave-kafka/src/main/java/smartthings/brave/kafka/producers/BaseTracingProducerInterceptor.java:53: error: cannot access Endpoint
      .remoteEndpoint(kafkaEndpoint);
      ^
  class file for zipkin.Endpoint not found
```

From what I can tell this shouldn't be happening but it doesn't seem like the compiler can differentiate the overloaded method `remoteEndpoint`, one of which takes the messing class.

Comment (adriancole) [reply brave-kafka/pom.xml]: yep. was a mistake to overload like this in hindsight. However, you can mark this dep optional, as it is only compile-required

Comment (llinder) [reply brave-kafka/pom.xml]: Cool thanks for pointing that out.  I'll update this to make it optional.

File: pom.xml [hunk 2]
```diff
@@ -69,8 +69,8 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <main.basedir>${project.basedir}</main.basedir>
-    <zipkin.version>2.0.0</zipkin.version>
-    <brave.version>4.8.0</brave.version>
+    <zipkin.version>2.4.5</zipkin.version>
+    <brave.version>4.14.1</brave.version>
     <brave-cassandra.version>0.3.0</brave-cassandra.version>
```
Comment (adriancole) [inline pom.xml]: let me bump this, to keep your graph sane

Comment (adriancole) [reply pom.xml]: https://github.com/openzipkin/brave-cassandra/pull/6

**Pull request merged**